<launch>
    <!-- start usb camera sensor    /-->
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <param name="video_device" value="/dev/video2" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap"/>
        <param name="camera_info_url" type="string" value="file://$(find visionbased_polishing)/yaml/cam_300_industry_20200518.yaml" />
    </node>
     <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
            <remap from="image" to="/usb_cam/image_raw"/>
            <param name="autosize" value="true" />
     </node>
    
</launch>
